components/i2c.rst

```diff
@@ -33,6 +34,7 @@ Configuration variables:
   Defaults to ``True``.
 - **frequency** (*Optional*, float): Set the frequency the I²C bus should operate on.
   Defaults to ``50kHz``. Values are ``50kHz``, ``100kHz``, ``200kHz``, ... ``800kHz``
+- **id** (Optional, ID): Manually specify the ID for this i2c Bus if you need multiple i2c buses.
```

Comment:
```suggestion
- **id** (*Optional*, :ref:`config-id`) Manually specify the ID for this I²C bus if you need multiple I²C buses.
```

Also add a small `.. note:` below for explaining how this is used on the receiving end (e.g. i2c_id)

Reply:
I was about to ask where I should put this.
Does it make sense to add this explanation to sensor core as it is applicable to at least all sensors that are i2c, spi or uart?

Reply:
No, sensor core is only about sensors.

Stuff like how multiple i2c buses can be used at once should go in the i2c page - that's where most users would look for first too.

Reply:
Does ESP8266 only support 1? I thought ESP8266 used a sw i2c implementation, so should allow multiple ones.

Reply:
From what i saw ESP32 can have 3 hw but i have only tested 2 so far.
I´m creating a new pull request with more details in a few minutes.
Any thing else i should include?

Reply:
I think your right and I can´t find the website I saw it before anymore.